.github/workflows/new_version_check.yml

```
@@ -25,7 +25,7 @@ jobs:
         run: |
           set -eo pipefail
           BASE_URL="https://versionhistory.googleapis.com/v1/chrome/platforms"
-          END_URL="channels/stable/versions/all/releases?filter=endtime=none,fraction>=0.5&order_by=version%20desc"
+          END_URL="channels/stable/versions/all/releases?filter=endtime=none,fraction%3E=0.5&order_by=version%20desc"
```

It's probably best to also encode some other bits here, sadly encoding is a [bit mess](https://url.spec.whatwg.org/#interface-urlsearchparams)

```suggestion
          END_URL="channels/stable/versions/all/releases?filter=endtime%3Dnone%2Cfraction%3E%3D0.5&order_by=version%20desc"
```